src/Forms/TextDisplay.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement, FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { fromTheme } from '../Theme';
+
+// An enum that represents the possible values for the label's positioning
+export enum POSITION {
+  TOP = 'top',
+  LEFT = 'left',
+}
+
+export interface TextDisplayProps {
+  /** The id of the label tied to this text display field */
+  id: string;
+  /** Specifies the label text */
+  label: string;
+  /** The name of the text display field */
+  name: string;
+  /** The current value in the text display field */
+  value: string;
+  /** Allows you to pass in a label position property from the POSITION enum
+   * @default LEFT
```

Comment:
This is a very minor point, but can we change the references of the default label position from LEFT to POSITION.LEFT? This is just to be a little more accurate in referring to the POSITION enum.

Reply:
Got it, wasn't sure what the right convention was here!